src/utils/handleTicketCreation.ts

```diff
@@ -249,6 +249,15 @@ const handleRest = async (
 
 			logsChannel.send({ embeds: [ticketEmbed] });
 		}
+
+		if (record.NotificationChannel !== '0') {
```

Comment:
This looks correct but can you add a case to set `NotificationChannel` to `0` in `channelDelete.ts` in the events folder?